src/constants.rs

```diff
@@ -93,8 +93,8 @@ pub const BASEPOINT_ORDER: Scalar = Scalar {
 
 use crate::ristretto::RistrettoBasepointTable;
 /// The Ristretto basepoint, as a `RistrettoBasepointTable` for scalar multiplication.
-pub const RISTRETTO_BASEPOINT_TABLE: RistrettoBasepointTable =
-    RistrettoBasepointTable(ED25519_BASEPOINT_TABLE);
+pub const RISTRETTO_BASEPOINT_TABLE: &'static RistrettoBasepointTable =
+    &RistrettoBasepointTable(ED25519_BASEPOINT_TABLE_INNER_DOC_HIDDEN);
```

Comment:
@elichai this in particular makes switching to `static` difficult... `RistrettoBasepointTable` is a newtype for an `EdwardsBasepointTable`.

At the very least it would need to have an added lifetime and refer to the `EdwardsBasepointTable` by reference. That's not entirely trivial as there are various APIs that expect ownership of the inner basepoint table, namely `RistrettoBasepointTable::create`

Reply:
nm, got it to work in d69e67c